src/convert.ts

```diff
@@ -0,0 +1,13 @@
+import { format, compareAsc } from "date-fns";
+
+export function convertSelicEntry(data: string, valor: string) {
+  try {
+    const date = new Date(format(format(data, "dd/MM/yyyy"), "yyyy-MM-dd"));
```

Comment:
Why not use [parse](https://date-fns.org/v3.6.0/docs/parse)?

```suggestion
    const date = parse(format(data, "dd/MM/yyyy");
```